runner/runner.go

```diff
@@ -59,17 +60,19 @@ func Start(ctx context.Context, config Config) error {
 	}
 	svc.registrationFailure.Store(types.Some(errors.New("not registered with ControlPlane")))
 
+	obs := observability.NewObservability()
```

Comment:
I would guess here is where we need to init the `obs.registrationFailure` but it was giving me some trouble :)